rtengine/iplocallab.cc

```diff
@@ -10872,6 +10872,16 @@ void ImProcFunctions::detail_mask(const array2D<float> &src, array2D<float> &mas
 // adapted to Rawtherapee Local adjustments J.Desmis january 2021
 //
 
+    static inline double igammalog    (double x, double p, double s, double g2, double g4)
+    {
+        return x <= g2 ? x / s : exp(log((x + g4) / (1.+ g4)) * p);//continuous
```

Comment:
This smells really expensive. I will have a look

Reply:
Mathematically this can also be `pow((x+g4)/(1.+g4),p)` right?

Reply:
right, or better (performance-wise) pow_F instead of pow